bundledata.go

```diff
@@ -364,9 +371,18 @@ func (verifier *bundleDataVerifier) verifyServices() {
 					verifier.addErrorf("invalid charm path in service %q: %v", name, err)
 				}
 			}
-		} else if _, err := ParseURL(svc.Charm); err != nil {
+		} else if curl, err = ParseURL(svc.Charm); err != nil {
 			verifier.addErrorf("invalid charm URL in service %q: %v", name, err)
 		}
+
+		// Check the series.
+		if curl != nil && svc.Series != "" {
+			verifier.addErrorf("service %q declares both a series and a charm store URL", name)
```

Comment:
"store" in the message may not be accurate in the future, if we add additional URL schemes